index.html

```diff
@@ -0,0 +1,137 @@
+            <div class="menu-close">
+              <img src="assets/img/Union.svg" alt="" />
+            </div>
+          </div>
+        </section>
+      </nav>
+    </div>
+    <div>
+      <section class="hello">
+        <h3 class="subheading">"Hello Micronauts!"</h3>
+        <h1 class="title">Microverse Concert Music for Developers 2021</h1>
+        <div class="intro">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Accus antium
+            ipsam iste itaque nihil architecto, volu tatem sed facilis porro
+            maiores, rerum vitae.
+          </p>
+        </div>
+        <p class="date">05(FRI)-07(SUN) DEC 2021</p>
+        <p class="at">@ YouTube: Microverse Official Channel</p>
+      </section>
+      <section class="program">
+        <h2>Main Program</h2>
+        <div class="program-wrap">
+          <div class="program-item">
+            <img src="assets/img/icon1.png" alt="" />
+            <h3>Live Music</h3>
+            <p>
+              More than 50 artists amet consectetur adipisicing elit. Repudiandae
+              harum nisi, aliquid.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon2.png" alt="" />
+            <h3>Visual arts</h3>
+            <p>
+              Trippy visuals from consectetur adipisicing elit. Repudiandae harum
+              nisi.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon3.png" alt="" />
+            <h3>Interviews</h3>
+            <p>
+              On stage with amet consectetur. Repudiandae harum nisi, aliquid
+              adipisicing elit.
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon4.png" alt="" />
+            <h3>20 Deejays</h3>
+            <p>
+              Turn on the volumen consectetur adipisicing elit. Repudiandae harum
+              nisi, aliquid sound!
+            </p>
+          </div>
+          <div class="program-item">
+            <img src="assets/img/icon5.png" alt="" />
+            <h3>Live Streaming</h3>
+            <p>
+              Watch it live sit adipisicing elit. Repudiandae harum nisi, aliquida
+              met consectetur.
+            </p>
+          </div>
+        </div>
+        <div class="program-link">
+          <a href="#" class="program-link">See the whole program</a>
+        </div>
+      </section>
+      <section class="featured" id="featured"></section>
+      <section class="partners">
+        <h2>Partners</h2>
+        <div class="partners-wrap">
+          <img src="assets/img/logo.png" alt="" />
+          <img src="assets/img/logo2.png" alt="" />
+          <img src="assets/img/logo3.png" alt="" />
+          <img src="assets/img/logo4.png" alt="" />
+          <img src="assets/img/logo5.png" alt="" />
+        </div>
+      </section>
+    </div>
```

Comment:
[OPTIONAL] It would be great also to wrap this section using a `main` tag, this will tell the reader what part of the document are they in.